pebble/src/main/java/com/mitchellbosecke/pebble/extension/core/Base64EncoderFilter.java

```diff
@@ -0,0 +1,37 @@
+package com.mitchellbosecke.pebble.extension.core;
+
+import com.mitchellbosecke.pebble.error.PebbleException;
+import com.mitchellbosecke.pebble.extension.Filter;
+import com.mitchellbosecke.pebble.template.EvaluationContext;
+import com.mitchellbosecke.pebble.template.PebbleTemplate;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class implements the 'base64encode' filter.
+ *
+ * @author Silviu Vergoti
+ */
+public class Base64EncoderFilter implements Filter {
```

Comment:
Can you add some documentation for the new filters too ? 

https://github.com/PebbleTemplates/pebble/tree/master/docs/src/orchid/resources

Reply:
Sure.